shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -0,0 +1,56 @@
+# platform = multi_platform_ol
+. /usr/share/scap-security-guide/remediation_functions
+populate var_accounts_authorized_local_users_regex
+
+# never delete the root user
+default_os_user="root"
+
+# add users sidamd and orasid
+userlist="root"
+
+# if /sapmnt/SID exists, add sidadm to the list
+SIDlist=$(find /sapmnt -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]$')
+for i in $SIDlist ; do
+	SID=${i:8:3}
+	userlist="$userlist|$(echo "$SID" | sed -e 's/\(.*\)/\L\1/')adm"
+done
```

Comment:
Your remediations are isolated tasks, I suggest that you make those tasks functions that you just invoke later.
One additional advantage is that name of the function may even make some comments redundant.